Gruntfile.js

```diff
@@ -191,7 +191,7 @@ module.exports = function(grunt) {
   grunt.registerTask("release:minor", ["bump:minor", "dist-compile", "gitcommit:version"]);
   grunt.registerTask("release:major", ["bump:major", "dist-compile", "gitcommit:version"]);
 
-  grunt.registerTask("dist-compile", ["test", "uglify", "compress"]);
+  grunt.registerTask("dist-compile", ["test", "exec:npm:sed-version", "uglify", "compress"]);
```

Comment:
I know, we're in a temporary bout of madness as we shift to npm as our build system. The real fix here is to just remove grunt completely but we're holding off in lieu of higher priority 3.0 stuff